.github/workflows/ci.yml

```
@@ -10,41 +10,22 @@ on:
-            echo "::group::Run simulation"
-            chmod +x tests/run_ODD_sim.sh
-            ./tests/run_ODD_sim.sh
-
-      - uses: actions/upload-artifact@v4
-        with:
-          name: sim_output
-          path: tests/run/
-          if-no-files-found: error
-          retention-days: 7
\ No newline at end of file
+    steps:
```

actionlint (1.7.4)

key "steps" is duplicated in "run" job. previously defined at line:14,col:5

(syntax-check)